loadmovr.py

```diff
@@ -84,38 +90,51 @@ def simulate_movr_load(conn_string, cities, movr_objects, active_rides, read_per
 
             if random.random() < read_percentage:
                 # simulate user loading screen
+                start = time.time()
```

Comment:
Nothing wrong with this API, an alternative pattern that I'm partial to would be doing something like
```
stop = stats.begin_measurement("get vehicles")
...
stop()
```
which adds a little more implementation complexity but means you don't have to put calls to `time.time()` everywhere you want to take a measurement.

(Don't feel any need to make this change, just floating it as an option.)